modules/ROOT/pages/dw-operators.adoc

```diff
@@ -363,3 +363,359 @@ xref:dataweave-variables.adoc#example_local_variable[Examples: Local DataWeave V
 | `if else` a| An `if` operator evaluates a conditional expression and returns the value under the `if` only if the conditional expression is true. Otherwise, it returns the expression under `else`. Every `if` expression must have a matching `else` expression. See xref:dataweave-flow-control.adoc#control_flow_if_else[if else] for an example.
 | `else if` | An `else` operator chains expressions together within an if-else construct by incorporating `else if`. See xref:dataweave-flow-control.adoc#control_flow_else_if[else if] for an example.
 |===
+
+== Update Operator
+
+DataWeave 2.0 supports the update operator that enables you to update a given field on a data structure with a new value. This use case is very common and was not simple to do in DataWeave.
+
+_Introduced in DataWeave 2.3.0. Supported by Mule 4.3 and later._ +
+
+Example: Using Update Operator, let's say that you have an object:
+[source,json,linenums]
+----
+{
+    "name": "Ken",
+    "lastName":"Shokida",
+    "age": 30
+}
+----
+
+And you want to update the age by increasing it with one, without the need of recreating all the object again. Using the current DataWeave syntax is:
```

Comment:
Note that `update` will be current in Mule 4.3. 

The "And ..." is a sentence fragment. Can you perhaps turn that into a complete sentence? 

"To increase the age value by one without using the `update` operator or recreating the object again, you can use the following DataWeave syntax:"

Reply:
I'd say perhaps we should mention something like `prior to 4.3, the way to accomplish this was...`

Reply:
Updated.